client/client.go

```diff
@@ -601,18 +602,23 @@ func (c *NetClient) ReadInNetworkFromPKI(pkiName string) error {
 	return nil
 }
 
-// NewClient constructor function to create an new client object.
-// Returns a new client object or an error, if occurred.
-func NewClient(id, host, port string, pubKey []byte, prvKey []byte, pkiDir string, provider config.MixConfig) (*NetClient, error) {
-	core := clientcore.NewCryptoClient(pubKey, prvKey, elliptic.P224(), provider, clientcore.NetworkPKI{})
+// The constructor function to create an new client object.
+// Function returns a new client object or an error, if occurred.
```

Comment:
Doesn't the change in comment formats break the linter?

Reply:
That's a very good question. For some reason my linter didn't raise any issues. I've already corrected the comment.